ccpp/suites/suite_FV3_GFS_v15_thompson_mynn_noah_flake.xml

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<suite name="FV3_GFS_v15_thompson_mynn_noah_flake" lib="ccppphys" ver="4">
```

Comment:
The syntax `lib="ccppphys" ver="4"` is obsolete (since several months already!) and needs to be replaced with just `version=1`.